frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-shell/readme.md

```diff
@@ -138,6 +142,49 @@ config>show-history 5
 List of invoked commands are persisted across UniConfig restarts and SSH connections.
 !!!
 
+### Unhide and Hide operations
+
+* Command 'unhide-get' is used to unhide attribute that was hidden in application properties for read.
+* Command 'unhide-set' is used to unhide attribute that was hidden in application properties for write.
+* Command 'hide-get' is used to hide again unhidden attribute with 'unhide-get' command.
+* Command 'hide-set' is used to hide again unhidden attribute with 'unhide-set' command.
+
+* When unhide is set for GET or SET operation, then request URL for this operation contains 'unhide' query parameter like bellow:  
+  http://localhost:8181/rests/data/network-topology:network-topology/topology=uniconfig/node=vnf21/configuration?unhide=all  
+  (In this example is unhide parameter set to 'all')
+
+!!!
+Command also confirm that attribute was added or removed from unhidden list.
+When is 'unhide-get' or 'hide-get' called without parameter, then output contains list of all unhidden parameters.
```

Comment:
When 'unhide-get' or 'hide-get' is called without a parameter, then the output contains a list ....